routers/likes.py

```
@@ -0,0 +1,31 @@
+    return {"success": "post_like"}
+
+
+# delete like
+@router.delete("/posts/{post_id}/likes")
+async def delete_like(post_id: PostId):
+    return {"success": "delete_like"}
+
+
+# check like status
+@router.get("/posts/{post_id}/likes")
+async def get_likes_status(post_id: PostId):
```

Ruff (0.14.11)

Unused function argument: `post_id`

(ARG001)